pulsar-broker-common/src/main/java/org/apache/pulsar/broker/resources/NamespaceResources.java

```diff
@@ -242,6 +242,12 @@ public PartitionedTopicResources(MetadataStore configurationStore, int operation
             super(configurationStore, PartitionedTopicMetadata.class, operationTimeoutSec);
         }
 
+        public void updatePartitionedTopic(TopicName tn, Function<PartitionedTopicMetadata,
```

Comment:
There is no need to add this method, because it seems that it only be called in the test.

My suggestion is that we could use `updatePartitionedTopicAsync().get()` instead of creating this new method

Reply:
@AnonHxy I have updated, PTAL.